main/src/cgeo/geocaching/log/LogEntry.java

```diff
@@ -45,7 +45,9 @@
     public final long date;
     /** Is a found log */
     public final int found;
-    /** Friend's log entry */
+    /** Own's or Friend's log entry indicator.
+     * Such logs will be visible in separated tab "Friends/Own Logs" in addition to main Logbook
+     * */
```

Comment:
While I appreciate these clarifications I'd rather see them in a separate PR as they, if I understand correctly, have nothing to do with the change as such.